_includes/includes/ui/keyboard-details.php

```diff
@@ -103,121 +92,31 @@ protected static function map_license($s) {
       return array_key_exists($s, $license_map) ? $license_map[$s] : $s;
     }
 
-    protected static function download_box($url, $title, $description, $class, $linktitle, $platform, $mode='standalone') {
-      $urlbits = explode('/', $url);
-      $filename = array_pop($urlbits);
-      $id = self::$id;
-      global $embed;
-
-      $e_filename = urlencode($filename);
-      $e_id = urlencode($id);
-
-      if($embed != 'none') {
```

Comment:
We're no longer considering the `$embed` state for this downloading aspect?  If we are, where is the logic?

Reply:
The embed and non-embed modes for downloading keyboards will be the same: we will request `/keyboards/install/<id>`, which the embedded dialog should capture and turn into a download request in the same way as the universal link handler. This does require changes to existing code.

Note that Windows, macOS and Linux 13.0 and earlier versions are already redirected to the legacy keyboard search so this should not impact them (they'll continue to use the old `keyman:` pattern).